gitapp/lib/users_list/home_page_view.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:gitapp/users_list/searchbar_widget.dart';
+import 'package:gitapp/users_list/connection_presenter.dart';
+import 'package:gitapp/users_list/user_cards_widget.dart';
+import 'package:gitapp/users_list/users_presenter.dart';
+import 'no_connection_widget.dart';
+
+class HomePage extends StatefulWidget {
+  HomePage({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  HomePageState createState() => HomePageState();
+}
+
+class HomePageState extends State<HomePage> with TickerProviderStateMixin {
+  final UsersPresenter users = UsersPresenter();
```

Comment:
It should be named Presenter not the user.

Reply:
Solved